### PR TITLE
Rookies

### PR DESCRIPTION
**Pull request by Gonaline**

### CODE REVIEW

**Review by RomainLF — APPROVED** [review]

**Comment by RomainLF** [comment]
ok for me

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: frontend/src/services/context/Ctx.jsx [hunk 1]
```diff
@@ -28,6 +29,14 @@ export function CtxProvider({ children }) {
 
   useEffect(() => {
     axios
+      .get(`${import.meta.env.VITE_BACKEND_URL}${"/team"}`)
+      .then(({ data }) => {
+        setTeam(data).then(() => {});
```
Comment (LoicBrassart) [inline frontend/src/services/context/Ctx.jsx:34]: Remve the empty `.then`

**Review by LoicBrassart — CHANGES_REQUESTED** [review]

**Comment by FrancoisSalmon** [comment]
supress PR replace by create pages nonProfessional & detail

**Pull request closed without merging**